org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/models/JythonGeneratorModel.java

```diff
@@ -0,0 +1,140 @@
+package org.eclipse.scanning.api.points.models;
+
+import java.util.ArrayList;
+import java.util.Collection;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Collection'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)